### DISCUSSION TITLE
Beginner User Unable to Fix Read Timed Out

### DISCUSSION THREAD

**Discussion by apl000** [iterate-ch/cyberduck#15397] [Q&A]
I've been using Cyberduck to upload to Backblaze B2 for a week now. Initially, I have no trouble uploading. But recently, I am unable to successfully complete my uploads.

I've managed to try changing the preferences as well to repeat failed networking tasks. And even though transfers says that upload is complete, the files are nowhere to be seen in the bucket. I have the knowledge of a 5 year old when it comes to the technical stuff so I am really having a hard time understanding the problem and how to fix it. 

![image](https://github.com/iterate-ch/cyberduck/assets/151890805/b90d3f37-b992-4424-8422-4ff35efb4a67)
![image](https://github.com/iterate-ch/cyberduck/assets/151890805/03076365-d434-4eff-a20e-718f114ff0e2)

**Comment by tatze96** [comment]
Duplicate for #15402
